cns/configuration/configuration.go

```diff
@@ -55,6 +55,7 @@ type CNSConfig struct {
 	TLSSubjectName              string
 	TelemetrySettings           TelemetrySettings
 	UseHTTPS                    bool
+	UseMTLS                     bool
```

Comment:
This will need to be added in CNS wrapper in Aquarius repo also

Reply:
created a PR for this in that repo